Library/Homebrew/cmd/tap.rb

```diff
@@ -3,7 +3,7 @@
 module Homebrew
   def tap
     if ARGV.empty?
-      puts Tap.names
+      Tap.each {|tap| puts " * [#{tap.get_priority.to_s.rjust(2, '0')}] #{tap.name}"}
```

Comment:
Seems the `rjust` bit should probably be part of a function on `tap`; `padded_priority` or something.